src/vopr.zig

```diff
@@ -436,6 +439,8 @@ pub const Simulator = struct {
     /// Total number of requests sent, including those that have not been delivered.
     /// Does not include `register` messages.
     requests_sent: usize = 0,
+    /// Total number of replies received by non-evicted clients.
+    /// Does not include `register` messages.
     requests_replied: usize = 0,
```

Comment:
Hm, do we now count `requests_replied` correctly? We update it `on_client_reply`, not `on_cluster_reply`, and it is at least non obvious why a reply won't get lost. I am thinking about the following scenario:

* client sends a request
* cluster processes it, sends a reply
* reply is lost
* a different client is registered, so the first one is evicted
* original client retries the request
* it gets an eviction message

Reply:
Note: vopr_lite considers this a success, as that's a liveness issue

Reply:
Hm, yeah, it actually is pretty obvious! I somehow managed to convince myself that it s not enough, but, no, this should be enough!